.spellcheck.yml

```diff
@@ -0,0 +1,20 @@
+matrix:
+- name: Markdown
+  aspell:
+    ignore-case: true
+    lang: en
+  dictionary:
+    wordlists:
+    - .wordlist.txt
+    encoding: utf-8
+  pipeline:
+  - pyspelling.filters.markdown:
+  - pyspelling.filters.html:
+      comments: false
+      ignores:
+      - code
+      - pre
+  sources:
+  - 'README.rst'
+  - 'docs/*.rst'
```

Comment:
i think this regex is an issue this will only search the top-level directory for `*.rst` but we may have more sub-directories. 

Can you just specify the directory so say `- docs`

Reply:
Hi @shahzebsiddiqui, neither docs nor docs/ work on my end. I add 'docs/**/*.rst' instead. Thank you!